### DISCUSSION TITLE
New member that is trying to get the beta with the Galactic Longitude fix

### DISCUSSION THREAD

**Discussion by Arcturios** [Stellarium/stellarium#2295] [General]
I have been using the Stellarium program back in January of last year,  but I am new to this site.

I was told that there was a beta that included fixes to galactic longitudinal coordinates.

Where do I go to access and download the beta for that?

I don't know what to do because I am new here.

I'd greatly appreciate any help.

This is a great astronomy program that I want to use to the maximum.
I am grateful for it being made and free to use.

**Comment by alex-w** [comment]
Please check https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Reply by Arcturios** [reply]
Thank you very much! I downloaded and it works perfectly.  I will spread the word about this great free program. I love it!  I'd even paid for it!
